### DISCUSSION TITLE
Nerdctl support squash image layers in containerd.

### DISCUSSION THREAD

**Discussion by lingdie** [containerd/nerdctl#3251] [Ideas]
Will nerdctl support squash image layers in containerd? This feature will be useful to slim images.

Ref: https://github.com/goldmann/docker-squash

**Comment by lingdie** [comment]
command examples:

```bash
# squash last 2 layers.
nerdctl squash docker.io/library/busybox:latest --from-layer 2 --tag docker.io/library/busybox:squash
```

**Comment by AkihiroSuda** [comment]
Duplicate:
- https://github.com/containerd/nerdctl/issues/3252
